source/content/wordpress-cron.md

```diff
@@ -1,56 +1,108 @@
 ---
 title: Cron for WordPress
-description: Configuring and optimizing the WP-Cron feature on your Pantheon WordPress site.
+description: Learn how to create and run jobs using Cron or WordPress's WP-Cron feature on your Pantheon site.
 cms: "WordPress"
 categories: [automate]
 tags: [cron]
+contributors: [whitneymeredith]
 ---
 
+## Cron for WordPress Overview
+
+Cron is a standard utility in Unix and Linux systems that it is used to schedule commands for automatic execution at configured intervals. These scheduled commands or tasks are known as Cron Jobs. The name Cron comes from the Unix system for scheduling jobs, ranging from once a minute to once a year. Cron is generally used for running scheduled backups, monitoring disk space, deleting files that are no longer required, running system maintenance tasks, and much more. WP-Cron must be disabled to allow Cron to schedule and run tasks.
+
 ## WP-Cron Overview
 
-WP-Cron executes specific tasks for WordPress powered sites. The name Cron comes from the Unix system for scheduling jobs, ranging from once a minute to once a year. Whether it's routine maintenance or scheduled alerts, any command that can be executed on Unix without user intervention can be scheduled as a Cron task.
+WP-Cron is a WordPress feature that only executes tasks when the page is loaded. WP-Cron is first loaded by WordPress when a page is requested on the front or backend of a site, at which point WP-Cron displays the necessary page to the site visitor. 
```

Comment:
Does WP-Cron run at the beginning of the page load, or the end? I would expect it would run at the end, after the page is rendered, as Drupal does.

In either event, I don't think that it is WP-Cron that is displaying the page to the visitor. I would expect that the normal rendering process would still be used, even when WP-Cron runs.